README.md

```diff
@@ -121,7 +121,7 @@ The following [official docker environment variables](https://docs.docker.com/en
 |-----------------|--------|-------------|-------------------------------------------------|
 | `DOCKER_CONFIG` | String | `~/.docker` | The location of your client configuration files |
 
-## Notes
+## Nodes
```

Comment:
While we just have a single entry `nodes output` in this section this is in fact "Notes" and not "Nodes". Same as https://github.com/docker/metadata-action#notes

Reply:
Thanks for the feedback, I wan't sure 😃